projects/collection-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -574,6 +581,14 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
     return this.editorService.contentPolicyUrl;
   }
 
+  showCommentAddedAgainstContent() {
```

Comment:
@pallakartheekreddy @rajnishdargan  we don't have to check if the creator and the current user id matches?

Reply:
not required @venkateshwarans. Because the creator only sees the draft version.